python/lsst/obs/base/test/compositeMapper.py

```diff
@@ -40,3 +40,10 @@ def _makeCamera(self, policy, repositoryDir):
         """Normally this makes a camera. For composite testing, we don't need a camera.
         """
         return None
+
+    def std_stdTestType(self, item, dataId):
```

Comment:
LDM-463 says method should look like `std_<datasetType>(self, item)` and this has one extra argument